frontend/drivers/platform_wii.c

```diff
@@ -176,6 +176,7 @@ void system_exec_wii(const char *_path, bool should_load_game)
    /* don't use memcpy, there might be an overlap. */
    memmove(EXECUTE_ADDR, dol, size);
    DCFlushRange(EXECUTE_ADDR, size);
+   free(dol);
```

Comment:
there is a warning about an overlap, so that free might cause a crash in case the information stored by malloc some bytes before `dol` is overwritten. this function will end up rebooting into a new clean environment anyway so the free isn't necessary.
I know this might seem a bit counter-intuitive, but not freeing the buffer here is better.